canvas_modules/common-canvas/__tests__/common-properties/components/table-toolbar-test.js

```diff
@@ -851,117 +901,127 @@ describe("TableToolbar multi select edit button works correctly", () => {
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableRows = tableUtilsRTL.getTableRows(summaryPanel);
 		tableRows.forEach((row) => {
-			const integerField = row.find("input").at(1);
-			expect(integerField.prop("value")).to.not.equal(100);
+			const integerField = within(row).getAllByRole("spinbutton")[0];
+			expect(integerField.value).to.not.equal("100");
 		});
 	});
 
 	it("should disable Edit button in selected rows when table toolbar has Edit button", () => {
-		let summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
+		const { container } = wrapper;
+		let summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
 
 		// select the first row in the table
-		let tableRows = tableUtils.getTableRows(summaryPanel);
+		let tableRows = tableUtilsRTL.getTableRows(summaryPanel);
 		expect(tableRows).to.have.length(4);
-		tableUtils.selectCheckboxes(summaryPanel, [0]);
-		summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableUtilsRTL.selectCheckboxes(summaryPanel, [0]);
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
 
 		// verify that the "Edit" button is not present in table toolbar because you selected only 1 row
-		let tableToolbar = wrapper.find("div.properties-table-toolbar");
-		let multiSelectEditButton = tableToolbar.find("button.properties-action-multi-select-edit");
-		expect(multiSelectEditButton).to.have.length(0);
+		let tableToolbar = container.querySelector("div.properties-table-toolbar");
+		let multiSelectEditButton = tableToolbar.querySelector("button.properties-action-multi-select-edit");
+		expect(multiSelectEditButton).to.not.exist;
 
 		// verify Edit button in ALL rows is enabled
-		tableRows = tableUtils.getTableRows(summaryPanel);
+		tableRows = tableUtilsRTL.getTableRows(summaryPanel);
 		tableRows.forEach((row) => {
-			const editRowButton = row.find("button.properties-subpanel-button");
-			expect(editRowButton.props()).to.have.property("disabled", false);
+			const editRowButton = row.querySelector("button.properties-subpanel-button");
+			expect(editRowButton.disabled).to.equal(false);
 		});
 
 		// select four rows in the table
-		tableUtils.selectCheckboxes(summaryPanel, [1, 2, 3]);
-		summaryPanel = propertyUtils.openSummaryPanel(wrapper, "SLE_mse_panel");
+		tableUtilsRTL.selectCheckboxes(summaryPanel, [1, 2, 3]);
+		summaryPanel = propertyUtilsRTL.openSummaryPanel(wrapper, "SLE_mse_panel");
 
 		// verify that the "Edit" button is present in table toolbar
-		tableToolbar = wrapper.find("div.properties-table-toolbar");
-		multiSelectEditButton = tableToolbar.find("button.properties-action-multi-select-edit");
-		expect(multiSelectEditButton).to.have.length(1);
+		tableToolbar = container.querySelector("div.properties-table-toolbar");
+		multiSelectEditButton = tableToolbar.querySelector("button.properties-action-multi-select-edit");
+		expect(multiSelectEditButton).to.exist;
```

Comment:
here can you add assertion to test if its only 1 just like enzyme ?

Reply:
Thanks. Could you please also add this wherever you are checking using `.to.exist` in all the test files ?